### ISSUE TITLE
Allow bulk deleting submissions in table view

### ISSUE THREAD

**Issue by pmusaraj** [kobotoolbox/kpi#1777]
Similar to bulk changing of validation status. 

@jnm I don't know whether we need backend work for bulk deleting.

**Comment by pmusaraj** [comment]
@alnf I created a separate issue for bulk uploading of XLSs: https://github.com/kobotoolbox/kpi/issues/1780

**Comment by magicznyleszek** [comment]
__NOTE: this should have a safe guard in a form of prompt - similar to project deletion.__

### Current situation

<img width="797" alt="screen shot 2018-08-25 at 13 56 01" src="https://user-images.githubusercontent.com/2521888/44618054-4d81f980-a86f-11e8-922c-9cabb292dd4f.png">

### Idea 1

Another button in the header. Fast and simple - could be problematic on smaller screens or if we want to add more header options in future.

<img width="801" alt="screen shot 2018-08-25 at 13 58 18" src="https://user-images.githubusercontent.com/2521888/44618060-54a90780-a86f-11e8-9c52-e6b41ce296e0.png">

### Idea 2

Change "Update selected" to "Manage…" dropdown and add "Delete all" as option. This makes it more future-friendly for adding more bulk options.

<img width="791" alt="screen shot 2018-08-25 at 14 00 14" src="https://user-images.githubusercontent.com/2521888/44618067-5e326f80-a86f-11e8-8dcd-88e3dbf68d5a.png">

### Idea 3

Move "Update selected" options to left, use "X selected" text as trigger button and add "Delete all" as option.

<img width="788" alt="screen shot 2018-08-27 at 11 06 19" src="https://user-images.githubusercontent.com/2521888/44651337-5a2a5d00-a9e9-11e8-82d1-20b4c1e6d3fb.png">

### Idea 4

Move "Update selected" options to unused table cell, use manage selected icon as trigger button and add "Delete all" as option.

<img width="790" alt="screen shot 2018-08-27 at 11 02 33" src="https://user-images.githubusercontent.com/2521888/44651385-8940ce80-a9e9-11e8-9149-12f227af7452.png">

**Comment by pmusaraj** [comment]
I like idea 3 the best. My only concern with idea 3 is that the text "x selected" isn't clear enough, but the blue color and the arrow icon indicate that it's an action item, with multiple options. So, that seems good enough. I also like that idea 3 frees up space.

**Comment by magicznyleszek** [comment]
@pmusaraj I wonder if those "Set status: …" options couldn't be named better. Or maybe we should leave current popover header?
<img width="183" alt="screen shot 2018-08-28 at 10 06 33" src="https://user-images.githubusercontent.com/2521888/44709492-1186a880-aaaa-11e8-926c-e0a95f68cc81.png">

**Comment by magicznyleszek** [comment]
Blocked by #1970

**Comment by magicznyleszek** [comment]
Merged into `two-databases` and will be released together
